sql-to-dbsp-compiler/lib/sqllib/src/lib.rs

```diff
@@ -889,6 +889,24 @@ some_polymorphic_function1!(sign, f, F32, F32);
 some_polymorphic_function1!(sign, d, F64, F64);
 some_polymorphic_function1!(sign, decimal, Decimal, Decimal);
 
+///////////////////// sin //////////////////////
```

Comment:
this many comments are overkill.
Just add one comment for all trigonometric functions